bin/calculate_TMB.py

```diff
@@ -0,0 +1,225 @@
+    parser = argparse.ArgumentParser(
+        description="Calculate TMB and plot AF distribution",
+        epilog="Example: python3 calculate_TMB.py input.tsv",
+    )
+    parser.add_argument(
+        "--file_in",
+	metavar="file_in",
+        type=Path,
+        help="Input tsv file of the vembrane TSV converter module",
+    )
+    parser.add_argument(
+        "--bedfile",
+        metavar="bedfile",
+        type=Path,
+        help="Path to the provided BED-file with .bed suffix.",
+    )
+    parser.add_argument(
+        "--panelsize_threshold",
+        metavar="panelsize_threshold",
+        type=int,
+        help="Expected minimal size of the BED-file before giving an error.",
+    )
+    parser.add_argument(
+        "--min_AF",
+        metavar="min_AF",
+        type=float,
+        help="Minimal AF threshold for the filtering procedure of the TMB module",
+    )
+    parser.add_argument(
+        "--max_AF",
+        metavar="max_AF",
+        type=float,
+        help="Maximal AF threshold for the filtering procedure of the TMB module",
+    )
+    parser.add_argument(
+        "--min_cov",
+        metavar="min_cov",
+        type=int,
+        help="Minimal coverage threshold for the filtering procedure of the TMB module",
+    )
+    parser.add_argument(
+        "--popfreq_max",
+        metavar="popfreq_max",
+        type=float,
+        help="Maximal prevalence AF in the gnomAD global population database",
+    )
+    parser.add_argument(
+        "--filter_muttype",
+        metavar="filter_muttype",
+        type=str,
+        help="Set the conditions for filtering, either selecting only SNVs, SNVs and MNVs or the whole dataset inlcuding InDels",
+    )
+    parser.add_argument(
+        "--file_out",
+        metavar="FILE_OUT",
+        type=Path,
+        help="Per Sample TMB value",
+    )
+    parser.add_argument(
+        "--plot_out",
+        metavar="PLOT_OUT",
+        type=Path,
+        help="Stacked histogramm of the AFs in the sample",
+    )
+    parser.add_argument(
+        "-l",
+        "--log-level",
+        help="The desired log level (default WARNING).",
+        choices=("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"),
+        default="WARNING",
+    )
+    return parser.parse_args(argv)
+
+logger = logging.getLogger()
+
+def preprocess_vembraneout(file_in, filter_muttype):
+    TMB_inputfile = pd.read_csv(file_in, sep="\t")
+    TMB_inputfile["Mut_ID"] = TMB_inputfile[["CHROM", "POS", "REF", "ALT"]].apply(lambda row: ':'.join(row.values.astype(str)), axis=1)
+    TMB_deduplicated = TMB_inputfile.drop_duplicates("Mut_ID", keep='first')
+    filtering_rates = []
+    filtering_rates.append(len(TMB_deduplicated['Mut_ID']))
+    ### reduce dataset on relevant columns for TMB calculation
+    TMB_minimal = TMB_deduplicated.filter(items =["Mut_ID", "CHROM", "POS", "REF", "ALT", "FILTER", "FORMAT_AF", "FORMAT_DP",  "CSQ_VARIANT_CLASS", "CSQ_Consequence", "CSQ_gnomADe_AF"], axis=1)
```

Comment:
My test runs failed because of the hard_coded `FORMAT_AF` as allele fraction column.
I fixed this in PR #28 and now the TSV header is always named "allele_fraction", regardless of the extraction/calculation method, which should be implemented in here before merging.